relayer/ops/monitoring/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build image
+
+FROM ubuntu:20.04 AS build
+
+# OS dependencies
+RUN apt-get update && apt-get install -y wget gcc build-essential git
+
+# Copy source
+
+RUN mkdir -p /starknet-monitoring/relayer
+COPY ./relayer /starknet-monitoring/relayer
+
+# Install golang
+
+RUN wget -c https://dl.google.com/go/go1.18.1.linux-amd64.tar.gz -O - \
+  | tar -xz -C /usr/local \
+  && mkdir -p /go/src /go/bin
+ENV PATH /usr/local/go/bin:$PATH
+
+# Compile binary
+
+WORKDIR /starknet-monitoring/relayer
+RUN go build -o ./monitoring ./cmd/monitoring/*.go
```

Comment:
Since all this is doing is just compiling go code, why not use the golang build image like core does?

https://github.com/smartcontractkit/chainlink/blob/b64fe315a2e7af10244f737c9acd7bd3c75bbeef/core/chainlink.Dockerfile#L30